e2e-tests/cypress/support/insurance/eligibility/submit-answers-happy-path.js

```diff
@@ -2,8 +2,15 @@ import submitInsuranceEligibilityAnswersFromBuyerCountryHappyPath from './submit
 
 import { completeStartForm, completeCheckIfEligibleForm } from './forms';
 
-export default () => {
-  completeStartForm();
-  completeCheckIfEligibleForm();
+/**
+ * startNewApplication defaults to false
+ * if true, means on the start new application page from check your answers which redirects to buyer country page
+ * if true, then do not need to complete start form or check if eligible
+ */
+export default (startNewApplication = false) => {
+  if (!startNewApplication) {
+    completeStartForm();
+    completeCheckIfEligibleForm();
+  }
```

Comment:
Do we have to make this change, can it be done in the E2E test itself?

Pretty confusing having this:


```js
cy.submitInsuranceEligibilityAnswersHappyPath(true);
````
To me it implies that you can pass "false" to _not_ submit as happy path, which doesn't make sense